language/move-lang/tests/move_check_testsuite.rs

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use move_lang::{move_check_no_report, shared::Address};
+use std::{fs, path::Path};
+use text_diff;
+
+use move_lang::test_utils::*;
+
+const OUT_EXT: &str = "out";
+const EXP_EXT: &str = "exp";
+
+const KEEP_TMP: &str = "KEEP";
+const UPDATE_BASELINE: &str = "UPDATE_BASELINE";
+
+fn format_diff(expected: String, actual: String) -> String {
+    use text_diff::*;
+    let (_, changeset) = diff(&expected, &actual, "\n");
+    let mut ret = String::new();
+
+    for seq in changeset {
+        match &seq {
+            Difference::Same(x) => {
+                ret.push_str(x);
+                ret.push_str("\n");
+            },
+            Difference::Add(x) => {
+                ret.push_str("\x1B[92m");
```

Comment:
You have `termcolor` in your list of dependencies, so wouldn't it be more robust you let it do the coloring? It's very unlikely hardcoded color codes will work on multiple platforms.

Reply:
Its only an indirect dependency. 

I will fix these colors another time